packages/dds/merge-tree/src/test/mergeTree.markRangeRemoved.deltaCallback.spec.ts

```diff
@@ -32,7 +32,7 @@ describe("MergeTree", () => {
             /* currentSeq: */ currentSequenceNumber);
     });
 
-    describe("markRangeRemoved", () => {
+    describe.only("markRangeRemoved", () => {
```

Comment:
```suggestion
    describe("markRangeRemoved", () => {
```

Reply:
oops. Out of curiosity does this repo have validation that this doesn't happen? Whiteboard took a nice approach here of using [this](https://www.npmjs.com/package/eslint-plugin-no-only-tests) in CI builds.